man/systemd-coredump.xml

```diff
@@ -352,6 +352,20 @@ flags:   ...
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>COREDUMP_PKGMETA_PACKAGE=</varname></term>
+        <term><varname>COREDUMP_PKGMETA_PACKAGEVERSION=</varname></term>
+        <term><varname>COREDUMP_PKGMETA_JSON=</varname></term>
+
+        <listitem><para>If the executable contained .package metadata ELF notes, they will be
+        parsed and attached. The <varname>package</varname> and <varname>packageVersion</varname>
+        of the 'main' ELF module (ie: the excutable) will be appended individually, and the
+        JSON-formatted content of each module will be appended as a single entry. For more
```

Comment:
Hmm, should we instead make one big json object:
```
{ 'module:fsverity': { ...},
  'module:foobar' : {... },
}
```
? This will be just less work for us, and less work for the consumer.

Reply:
Ok, done:

```
 COREDUMP_PKGMETA_PACKAGE=fsverity-utils
 COREDUMP_PKGMETA_PACKAGEVERSION=1.3-1
 COREDUMP_PKGMETA_JSON={"/home/bluca/git/fsverity-utils/libfsverity.so.0":{"packageType":"deb","package":"fsverity-utils","packageVersion":"1.3-1","buildid":"fa40fdfb79aea84167c98ca8a89add9ac4f51069"},"fsverity":{"packageType":"deb","package":"fsverity-utils","packageVersion":"1.3-1","buildid":"ac89bf7175b04d7eec7f6544a923f45be111f0be"}}
```